crates/iota-genesis-builder/src/lib.rs

```diff
@@ -820,6 +871,8 @@ impl Builder {
             genesis_stake: Default::default(),
             migration_sources,
             migration_tx_data,
+            delegator: None,     // todo: Probably need to load the delegator?
+            delegator_map: None, // todo: Probably need to load the delegator_map?
```

Comment:
Yes, check the comment to the `save()`  below. For delegator map you find the `from_csv`.